plugins/bazaar/README.md

```diff
@@ -76,17 +76,17 @@ export const homePage = (
        <Grid container spacing={3}>
 
 +       <Grid item xs={12} md={6}>
-+         <BazaarOverviewCard variant={'latest'} limit={4} />
++         <BazaarOverviewCard order={'latest'}/>
```

Comment:
you may want to encourage `order="latest"` here and below, since it looks more like html - no need for the curly braces if it's a string